common/metrics/defs.go

```diff
@@ -1712,6 +1712,8 @@ const (
 	ParentClosePolicyProcessorSuccess
 	ParentClosePolicyProcessorFailures
 
+	CadenceErrWorkflowExecutionAlreadyCompletedCounter
```

Comment:
nit: group with other `CadenceErrXXXCounter` metrics?

Reply:
I thought about that but wasn't sure how the enums were stored in the metrics; putting this up there might change the enum values of the other counters and metrics might show up wrong in the dashboards. Do you know if it's okay to put new enums in between the other ones?

Reply:
yes, it's ok. Those enums will be mapped to strings.